lib/features/create/create_replace_textures/create_replace_textures_viewmodel.dart

```diff
@@ -269,6 +264,10 @@ Future<bool> processFile(String fileName, Uint8List data, String outputPath, Fun
       final texture = Texture.empty();
       texture.open(data);
 
+      if(texture.gameVersion == Version.roy){
```

Comment:
what is this doing? why roy?

Reply:
Those versions are stupid names someone put, roy is version 1, which is the texture version that is for HD textures

Reply:
could you add a comment above this explaining that? something as simple as
```
// don't try to extract hd textures
```